### PR TITLE
Fix issue with multiline erros and move last n reports to base report class

### PR DESCRIPTION
**Pull request by dkropachev**
Fix issue with multiline erros and move last n reports to base report class
https://trello.com/c/f8RRXLdZ/1761-fix-events-list-in-the-longevity-email
https://trello.com/c/HLHkxrnq/1749-events-summary-list-shoud-be-reused-in-all-the-email-templates

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: sdcm/results_artifacts.html [hunk 1]
```diff
@@ -60,6 +60,20 @@ <h3>
         </ul>
     </div>
 
+    {% if last_events %}
```
Comment (fruch) [inline sdcm/results_artifacts.html]: can we move this to results_base.html ? as a block ?
```
# results_base.html
{% block events_summary %}
..everything here..
{% endblock %}
```

and other type would be able to overwrite it if needed (or add more sub blocks for specific things)

**Comment by dkropachev** [comment]
Gemini, Longeviti and microbenchmarks tested locally

**Comment by roydahan** [comment]
@dkropachev can you please share an example of how it looks?

**Comment by dkropachev** [comment]
New reports
[new_reports.zip](https://github.com/scylladb/scylla-cluster-tests/files/4343636/new_reports.zip)

**Comment by bentsi** [comment]
The email looks better. Improvements:
1. Remove the "Events summary" we don't need it anymore. Instead, add a number to the severity that  is shown in the "last events"
example:
image
2. let's remove "Last events" 
3. show full event in the severities
4.decrease number of events shown from 5 to 3

**Comment by roydahan** [comment]
> The email looks better. Improvements:
> 
> 1. Remove the "Events summary" we don't need it anymore. Instead, add a number to the severity that  is shown in the "last events"
>    example:
>    image
I disagree, the event summary is good enough like this, if we will have too many events or something like that the require scrolling, it's always good to see the numbers above.
> 2. let's remove "Last events"
I agree if they are part of the events part, however in the example @dkropachev shared, I seed that the last events contain different events than the section below, why?
> 3. show full event string in the severities
Agree
>    4.decrease number of events shown from 5 to 3
Why? I prefer to see more than less.

**Comment by bentsi** [comment]
@roydahan regarding the counter of events, the purpose is to save space and not to duplicate information, I meant the following:
image
the same is for decreasing the num of events. 
The reason for decreasing is because GMAIL is clipping email if it exceeds 102k.

**Comment by dkropachev** [comment]
@roydahan, @bentsi ,
Please check reports after the change. 
[new_reports.zip](https://github.com/scylladb/scylla-cluster-tests/files/4367380/new_reports.zip)

**Comment by dkropachev** [comment]
> > let's remove "Last events"
> I agree if they are part of the events part, however in the example @dkropachev shared, I seed that the last events contain different events than the section below, why?
We were collection first 100 critical events for that section and 5 last one for last events seciont.
Second problem is that data is forged, since i'am practicing on old reports.

**Pull request merged**